server/models/user.js

```diff
@@ -25,8 +25,13 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false
     },
+    role: {
+      type: DataTypes.ENUM('superadmin', 'admin', 'user'),
+      allowNull: false,
+      defaultValue: 'user'
+    },
     social: {
-      type: DataTypes.ENUM('facebook', 'twitter', 'google', null),
+      type: DataTypes.STRING,
```

Comment:
Why this change to `STRING` and how does it affect the role based access control feature?

Reply:
It seems sequelize has an issue with having two ENUM columns on the same table, hence the change